tests/phpunit/integration/api/image/beans-image-editor/run.php

```diff
@@ -40,6 +41,16 @@ public static function setUpBeforeClass() {
 		static::$fixtures_dir = realpath( __DIR__ . '/../fixtures' );
 	}
 
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	public function setUp() {
```

Comment:
Remove this method as it will no longer be needed.  Why? We are moving the mock for `wp_normalize_path` to the base test case.